platform/ios/MGLMapboxEvents.m

```diff
@@ -16,6 +16,7 @@
 static const NSUInteger version = 1;
 static NSString *const MGLMapboxEventsUserAgent = @"MapboxEventsiOS/1.0";
 static NSString *MGLMapboxEventsAPIBase = @"https://api.tiles.mapbox.com";
+static BOOL usingTestServer = NO;
```

Comment:
Globals should be named according to the style seen above: MGLMapboxEventsUsesTestServer.